streaming/src/test/scala/org/apache/spark/streaming/ReceivedBlockHandlerSuite.scala

```diff
@@ -251,9 +377,20 @@ class ReceivedBlockHandlerSuite
     (blockIds, storeResults)
   }
 
+  /** Store block using a handler */
+  private def storeBlock(
```

Comment:
Instead of adding this, please use the other `storeBlocks`. there is already semantics here that randomized blockIds etc to prevent any conflicts due to the same block id being used. So best to use this rather than use the same block id every time and mask errors. 

Also use the returned block Id to clean up the block later, in `testRecordCount`